libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -85,22 +85,19 @@ class ESP8266WiFiSTAClass {
 
     protected:
 
-      static bool _useStaticIp;
+        static bool _useStaticIp;
 
     // ----------------------------------------------------------------------------------------------
     // ------------------------------------ STA remote configure  -----------------------------------
     // ----------------------------------------------------------------------------------------------
 
+    protected:
+    
+        static void WPSStatusCB(wps_cb_status status);

```

Comment:
This method shouldn't be part of the class, because it is private to the internal code (see comment above).

Reply:
It uses the same model as what we have in for example WiFiClient about the lwIP's C-style callbacks (there's no context pointer with WPS though).